tools/biolatency_example.txt

```diff
@@ -292,11 +292,62 @@ flags = Priority-Metadata-Write
 These can be handled differently by the storage device, and this mode lets us
 examine their performance in isolation.
 
+The -j option prints a dictionary of the histogram.
+In most cases, this flag will be used with the -f flag. For example:
+
+# ./biolatency.py -j
+Tracing block device I/O... Hit Ctrl-C to end.
+^C
+{32: 4, 2: 0, 4096: 32, 4: 0, 1024: 10, 8192: 0, 8: 0, 64: 38, 128: 42, 256: 5, 16: 0, 512: 1, 2048: 0}
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
```

Comment:
Look at bpftool json output example
   https://manpages.ubuntu.com/manpages/focal/man8/bpftool-prog.8.html
can we design something similar?

If we want to output to json, we should just output in json format, there is no need to output old way format. So the above format maybe looks like
  [{
     "usecs-start": 0,
     "usecs-end" : 1,
     "count" : 4096
   },{
     "usecs-start": 2,
     "usecs-end": 3,
     "count": 0
    } ...
  }

The above bpftool example is with pretty print. I guess your json output does not need to be pretty print.

I think we do not need "-f" option. json format is mostly for automatic/advanced processing, we do not need to display banner "Tracing block device I/O... Hit Ctrl-C to end." so the output can be redirected to a file if needed. In man page, we can mention "-j" option will omit banner to formatted output. WDYT?

Reply:
Yes, I think that is a better design. Pushing that change soon. Thanks!